eslint-local-rules/disallowZoneJsPatchedValues.js

```diff
@@ -13,7 +13,10 @@ const PROBLEMATIC_IDENTIFIERS = {
   setInterval: 'Use `setInterval` from @datadog/browser-core instead',
   clearInterval: 'Use `clearInterval` from @datadog/browser-core instead',
 
-  // TODO: disallow addEventListener, removeEventListener
+  // Using the patched `addEventListener` from Zone.js might trigger a memory leak in Firefox, see
+  // PR #1860
+  addEventListener: 'Use `addEventListener` from @datadog/browser-core instead',
+  removeEventListener: 'Use `addEventListener` from @datadog/browser-core instead',
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** I thought it was a typo at first 😅 maybe something like that could be helpful:‏
```suggestion
  removeEventListener: 'Use `addEventListener().stop` from @datadog/browser-core instead',
```